websites/S/Stack Exchange/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.5",
+    "author": {
+      "name": "AcidEpicice",
+      "id": "249992970235936768"
+    },
+    "url": ["stackexchange.com", "serverfault.com", "askubuntu.com", "superuser.com"],
+    "regExp": "(.*\\.)*stackexchange\\.com|superuser\\.com|askubuntu\\.com|serverfault\\.com",
```

Comment:
this seems rather redundant, unless you want to match all of those domains with anything before them, in which case the regex is incorrect
```suggestion
    "regExp": "(.*\\.)*stackexchange\\.com",
```

Reply:
Sorry, could you elaborate what the issue with the regex is? If you mean that I'm incorrectly catching subdomains, I _do_ want to catch all subdomains of the listed sites

Reply:
This regex will match `ggerhgyuierhbgre.stackexchange.com` but not `ergergerger.serverfault.com`, which is what you want

Reply:
Looks like it does, actually?
![image](https://user-images.githubusercontent.com/38502391/146091122-9861702f-90f2-4901-9c20-696cab08b82b.png)

Reply:
It shouldn’t then, here’s the correct one (do test this though)
```suggestion
    "regExp": "$(.*\\.)*(stackexchange\\.com|superuser\\.com|askubuntu\\.com|serverfault\\.com)^",
```